atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -135,85 +180,20 @@ public static SerializableTransactionManager createInMemory(Schema schema) {
      * {@link com.palantir.atlasdb.keyvalue.impl.InMemoryKeyValueService}. This should be used for testing
      * purposes only.
      */
-    public static SerializableTransactionManager createInMemory(Set<Schema> schemas) {
-        AtlasDbConfig config = ImmutableAtlasDbConfig.builder()
-                .keyValueService(new InMemoryAtlasDbConfig())
-                .build();
-        return create(config,
-                java.util.Optional::empty,
-                schemas,
-                x -> { },
-                false);
+    static SerializableTransactionManager createInMemory(Set<Schema> schemas) {
+        AtlasDbConfig config = ImmutableAtlasDbConfig.builder().keyValueService(new InMemoryAtlasDbConfig()).build();
+        return builder().config(config).schemas(schemas).buildSerializable();
     }
 
-    /**
-     * Create a {@link SerializableTransactionManager} with provided configurations, {@link Schema},
-     * and an environment in which to register HTTP server endpoints.
-     */
-    public static SerializableTransactionManager create(
```

Comment:
Can we keep these methods around, but mark them as deprecated? Seems nicer to me than breaking the world, and allows us to add some kind of lotion to the bandaid before ripping it off - with some time, the bandaid might fall painlessly off on its own.

(OK, I think I've taken the metaphor far enough)